lib/generator.js

```diff
@@ -2026,6 +2024,7 @@
             
     exports.Generator         = Generator;
     exports.createGenerator   = createGenerator;
+    exports.logStream         = _logStream;
```

Comment:
Maybe `_logStream` should just be renamed to `logStream` because it is no longer private?